src/maliput_malidrive/builder/road_curve_factory.cc

```diff
@@ -274,6 +284,44 @@ std::unique_ptr<malidrive::road_curve::Function> RoadCurveFactory::MakeLaneWidth
     }
     polynomials.emplace_back(MakeCubicPolynomial(coeffs[3], coeffs[2], coeffs[1], coeffs[0], p0_i, p1_i));
   }
+  return polynomials;
+}
+
+bool RoadCurveFactory::AreLaneWidthsContinuous(const xodr::LaneWidth& a, const xodr::LaneWidth& b, double b_end) const {
+  const auto left = MakeCubicPolynomial(a.d, a.c, a.b, a.a, 0., b.s_0 - a.s_0);
+  const auto right = MakeCubicPolynomial(b.d, b.c, b.b, b.a, 0., b_end);
```

Comment:
nit: I understand the case, but what's left and what's right? :smiley: Use better variable names

Reply:
Done. LMK if the changes make it clearer though.